vtex/omnishipping
# Fixed Logistics info hidration for second purchase

#### What is the purpose of this pull request?
Fixed Logistics info hidration for second purchase

#### How should this be manually tested?
1. Add [items to cart](https://fernando--automacaoqa.myvtex.com/checkout/cart/add?&workspace=fernando&sku=2000002&qty=1&seller=1&sc=1);
2. Identify as `felipe.nascimento@vtex.com`;
3. It should go directly to payment.
#### Screenshots or example usage

#### Types of changes
